corporate/urls.py

```diff
@@ -233,8 +233,8 @@
         name="remote_server_event_status_page",
     ),
     # Remote variants of above API endpoints.
-    path("json/realm/<realm_uuid>/sponsorship", remote_realm_sponsorship),
-    path("json/server/<server_uuid>/sponsorship", remote_server_sponsorship),
+    path("json/realm/<realm_uuid>/billing/sponsorship", remote_realm_sponsorship),
+    path("json/server/<server_uuid>/billing/sponsorship", remote_server_sponsorship),
```

Comment:
These changes were supposed to be in the previous commit, right?

Reply:
Done.